pkg/platform/platform.go

```diff
@@ -30,6 +30,7 @@ var cloudProviderConfig = types.NamespacedName{
 var (
 	masterRoleMachineConfigLabel = map[string]string{"machineconfiguration.openshift.io/role": "master"}
 	workerRoleMachineConfigLabel = map[string]string{"machineconfiguration.openshift.io/role": "worker"}
+	EWIPsecOnly                  = true
```

Comment:
Why can't this be a constant?

Also add a comment explaining what this does and intentions. I guess you are doing it this way to have an easy way to revert in the future. But I am only guessing, it would be nice to have a comment explaining.